src/stdlib_bitsets.fypp

```diff
@@ -0,0 +1,2126 @@
+
+        elemental module subroutine clear_bit_large(self, pos)
+!! Version: experimental
+!!
+!! Sets to zero the bit at `pos` position in `self`. If 'pos` is less than
+!! zero or greater than `bits(self)-1` it is ignored.
+            class(bitset_large), intent(inout) :: self
+            integer(bits_kind), intent(in)     :: pos
+        end subroutine clear_bit_large
+
+        pure module subroutine clear_range_large(self, start_pos, stop_pos)
+!! Version: experimental
+!!
+!! Sets to zero all bits from the `start_pos` to `stop_pos` positions in `self`.
+!! If `stop_pos < start_pos` then no bits are modified. Positions outside
+!! the range 0 to `bits(set)-1` are ignored.
+            class(bitset_large), intent(inout) :: self
+            integer(bits_kind), intent(in)     :: start_pos, stop_pos
+        end subroutine clear_range_large
+
+        elemental module subroutine flip_bit_large(self, pos)
+!! Version: experimental
+!!
+!! Flips the bit value at the `pos` position in `self`, provided the position is
+!! valid. If `pos` is less than 0 or greater than `bits(self)-1`, no value is
+!! changed.
+            class(bitset_large), intent(inout) :: self
+            integer(bits_kind), intent(in)     :: pos
+        end subroutine flip_bit_large
+
+        pure module subroutine flip_range_large(self, start_pos, stop_pos)
+!! Version: experimental
+!!
+!! Flips all valid bits from the `start_pos` to the `stop_pos` positions in
+!! `self`. If `stop_pos < start_pos` no bits are flipped. Positions less than
+!! 0 or greater than `bits(self)-1` are ignored.
+            class(bitset_large), intent(inout) :: self
+            integer(bits_kind), intent(in)     :: start_pos, stop_pos
+        end subroutine flip_range_large
+
+        module subroutine from_string_large(self, string, status)
+!! Version: experimental
+!!
+!! Initializes the bitset `self` treating `string` as a binary literal
+!! `status` may have the values:
+!! * `success` - if no problems were found,
+!! * `alloc_fault` - if allocation of the bitset failed
+!! * `char_string_too_large_error` - if `string` was too large, or
+!! * `char_string_invalid_error` - if string had an invalid character.
+            class(bitset_large), intent(out) :: self
+            character(*), intent(in)         :: string
+            integer, intent(out), optional   :: status
+        end subroutine from_string_large
+
+        module subroutine init_zero_large(self, bits, status)
+!! Version: experimental
+!!
+!! Creates the bitset, `self`, of size `bits`, with all bits initialized to
+!! zero. `bits` must be non-negative.  If an error occurs and `status` is
+!! absent then processing stops with an informative stop code. `status`
+!! will have one of the values;
+!! * `success` -  if no problems were found,
+!! * `alloc_fault` - if memory allocation failed
+!! * `array_size_invalid_error` - if `bits` is either negative or larger
+!!   than 64 with `self` of class `bitset_64`, or
```

Comment:
This sentence is not finished, right?

Reply:
As near as I could tell the dangling ", or" was inappropriate and my revisions have replaced it with a "."